### PR TITLE
[Term Entry] JavaScript Dates: .toDateString()

### PR DESCRIPTION
**Pull request by yeipkwa**
### Description

- Added toDateString() term for JavaScript Dates.
- Closes issue #3995 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+Title: 'toDateString()'
+Description: 'returns a string representing the date portion of a Date instance'
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:3]:
```suggestion
Description: 'Returns a string representing the date portion of a Date instance' 
```

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`toDateString()`** method of Date instances returns the date portion of a Date object as a string.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:15]: Make this a bit more descriptive.

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Syntax
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:17]:
```suggestion
## Syntax

```

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+```pseudo
+const d = new Date('January 1, 2000 00:00')
+d.toDateString()
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:20]:
```suggestion
dateObject.toDateString()
```

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## Example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:23]:
```suggestion
## Example

```

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+```js
+const date = new Date("December 5, 2023 8:53")
+let  dateString= date.toDateString();
+console.log(dateString)
+// Output: Tue Dec 05 2023
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:28]:
```suggestion
```
Add a output block below this code snippet with shell tag. Like this:
```shell
output
```
image

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+The **`toDateString()`** method of Date instances returns the date portion of a Date object as a string.
+
+## Syntax
+```pseudo
+const d = new Date('January 1, 2000 00:00')
+d.toDateString()
+```
+
+## Example
+```js
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:24]: add a small introductory sentence above this code snippet which can act as a path for author to get ready and what to expect from the code snippet below.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have few comments for you to work on @yeipkwa

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md (hunk 2 above)
Comment (yeipkwa) [reply content/javascript/concepts/dates/terms/toDateString/toDateString.md:15]: Hi, thanks for the feedback. I'm not sure I understand what you mean by "more descriptive". Is there a specific way you would like me describe the method?

Comment (dakshdeepHERE) [reply content/javascript/concepts/dates/terms/toDateString/toDateString.md:15]: @yeipkwa sure. So more descriptively, I wanted you to add more info about `toDateString()`. For example: When applied to a Date object, `toDateString()` retrieves the date part of the object's value and formats it as a string representing the date in a locale-specific format. We can add this sentence to this sentence too.

Comment (dakshdeepHERE) [reply content/javascript/concepts/dates/terms/toDateString/toDateString.md:15]: Apart from this comment please check out the other comments and work on them too. 🚀

Comment (yeipkwa) [reply content/javascript/concepts/dates/terms/toDateString/toDateString.md:15]: Okay, got it. Thank you!

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+## Syntax
+
+```pseudo
+dateObject.toDateString()
+```
+
+## Example
+
+In this example, a new instance of the date object is created and stored in the variable, `date`. The toDateString()method is then called on `date`, and the string returned is stored in `dateString` and printed out on the console.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:25]:
```suggestion
In this example, a new instance of the date object is created and stored in the variable, `date`. The `.toDateString()` method is then called on `date`, and the string returned is stored in `dateString` and printed out on the console:
```

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+```js
+const date = new Date("December 5, 2023 8:53")
+let  dateString= date.toDateString();
+console.log(dateString)
+```
+
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:32]:
```suggestion
The above example will give the following output:

```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few more changes

**Review by dakshdeepHERE — APPROVED** [review]
This look good for second review 🚀

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+---
+Title: 'toDateString()'
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:2]:
```suggestion
Title: '.toDateString()'
```

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+When applied to a Date object, **`toDateString()`** retrieves the date portion of the object's value, and returns a string representation of this date in a locale-specific format.
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:15]:
```suggestion
When applied to a Date object, **`.toDateString()`** retrieves the date portion of the object's value, and returns a string representation of this date in a locale-specific format.
```

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+## Syntax
+
+```pseudo
+dateObject.toDateString()
+```
+
+## Example
+
+In this example, a new instance of the date object is created and stored in the variable, `date`. The toDateString() method is then called on `date`, and the string returned is stored in `dateString` and printed out on the console:
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:25]:
```suggestion
In this example, a new instance of the date object is created and stored in the variable, `date`. The `toDateString()` method is then called on `date`, and the string returned is stored in `dateString` and printed out on the console:
```
Method names or reserved words should be enclosed in the backticks.

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+The above example will give the following output:
+
+```shell
+Tue Dec 05 2023
+```
+
+## Codebyte Example
+
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:40]: Write a line saying the code block is runnable and the reader can execute it with their inputs.

File: content/javascript/concepts/dates/terms/toDateString/toDateString.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/dates/terms/toDateString/toDateString.md:41]:
```suggestion
```codebyte/javascript
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@yeipkwa Here are some changes from my side!!

**Review by avdhoottt — APPROVED** [review]
Looks good to get merged!!

**Pull request merged**